packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -11,12 +13,19 @@ import { connectStore, formatSize, getXoaPlan } from 'utils'
 import { createGetObjectsOfType } from 'selectors'
 import { downloadAndInstallResource, deleteTemplates } from 'xo'
 import { error, success } from 'notification'
-import { find, filter } from 'lodash'
+import { find, filter, isEmpty, map, omit, startCase } from 'lodash'
 import { injectState, provideState } from 'reaclette'
 import { withRouter } from 'react-router'
 
 import ResourceForm from './resource-form'
 
+const Li = props => <li {...props} className='list-group-item' />
+const Ul = props => <ul {...props} className='list-group' />
+
+const BANNED_FIELDS = ['any', 'description'] // These fields will not be displayed on description modal
```

Comment:
What's `'any'`?

Reply:
A default property for public property https://gitlab.com/vates/www-xo/blob/develop/server/resource/index.json#L151

Reply:
Why is it necessary? Is it in the resources documentation?

Reply:
It's not necessary, it's just a part of old resources... and it's not in the resources documentation.

`any` means that we can just add any additional data

Reply:
I don't understand why a resource metadata would contain a `any` property. But if it's the case, then we need to understand what it is in XO otherwise we'll end up removing it for sure the next time we change this code. So please add a small comment to explain that as well.